alns/accept/tests/test_adaptive_threshold.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np
+import numpy.random as rnd
+from numpy.testing import assert_, assert_equal, assert_raises
+from pytest import mark
+
+from alns.accept.AdaptiveThreshold import AdaptiveThreshold
+from alns.tests.states import One, Two, VarObj, Zero
+
+
+@mark.parametrize(
+    "eta, gamma",
+    [
+        (-1, 3),  # eta cannot be < 0 or > 1
+        (0.5, -2),  # gamma cannot be < 0
```

Comment:
Can we add another test that checks $\eta > 1$? These cases only check $\eta \in \lbrace -1, 0.5 \rbrace$.